### DISCUSSION TITLE
Use Gradle instead of Maven

### DISCUSSION THREAD

**Discussion by ohonchar** [Cornutum/tcases#280] [General]
Hi, I would like to use your Tcases, but our company use Gradle as main builder. Is it possible to implement your tool with Gradle configurations? Thanks in advance

**Comment by kerrykimbrough** [comment]
I have no experience with Gradle. Perhaps you or someone else would like to contribute this to the project?

**Reply by ohonchar** [reply]
Ok. Why not to try. I would like to investigate this.
